util/testutil/dockerd/client/types/types.go

```diff
@@ -0,0 +1,13 @@
+package types
```

Comment:
I don't think this types package adds much value. `ErrorResponse` seems to be the only thing somewhat in use and can be just in the `client` package.